lib/split/algorithms/systematic_sampling.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+module Split
+  module Algorithms
+    module SystematicSampling
+      def self.choose_alternative(experiment)
+        count = experiment.next_cohorting_block_index
```

Comment:
This is the Redis call to increment a counter that keeps track of the number of users that have been run through this algorithm so far.

It will return an integer from Redis that will be the next value of the counter, because this value is supplied by Redis there shouldn't be any race conditions where multiple users have the same `count` variable

Reply:
`next_cohorting_block_index` is more like a cohort count? The `index` leads me to a misunderstanding, it's the index number in a block, which shouldn't be larger than the size of the block.